A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/A32NX_BaseNDCompass.js

```diff
@@ -447,38 +447,39 @@ class Jet_NDCompass extends HTMLElement {
-            if (this.navigationMode == Jet_NDCompass_Navigation.ILS || this.navigationMode == Jet_NDCompass_Navigation.VOR) {
-                this.courseGroup.classList.toggle('hide', false);
-                const compass = Number(this.getAttribute('rotation'));
-                let displayCourseDeviation = false;
-                let displayVerticalDeviation = false;
-                if (this.navigationMode == Jet_NDCompass_Navigation.ILS || this.navigationMode === Jet_NDCompass_Navigation.VOR) {
-                    let beacon;
-                    if (this.navigationMode == Jet_NDCompass_Navigation.ILS) {
-                        beacon = this.gps.radioNav.getBestILSBeacon();
-                    } else if (this.navigationMode === Jet_NDCompass_Navigation.VOR) {
-                        beacon = this.gps.radioNav.getBestVORBeacon();
-                    }
-                    if (beacon.id > 0) {
-                        displayCourseDeviation = true;
-                        let deviation = (SimVar.GetSimVarValue("NAV CDI:" + beacon.id, "number") / 127);
-                        const backCourse = SimVar.GetSimVarValue("AUTOPILOT BACKCOURSE HOLD", "bool");
-                        if (backCourse) {
-                            deviation = -deviation;
-                        }
-                        this.setAttribute("course", beacon.course.toString());
-                        this.setAttribute("course_deviation", deviation.toString());
-                        if (SimVar.GetSimVarValue("NAV HAS GLIDE SLOPE:" + beacon.id, "Bool")) {
-                            displayVerticalDeviation = true;
-                            this.setAttribute("vertical_deviation", (SimVar.GetSimVarValue("NAV GSI:" + beacon.id, "number") / 127.0).toString());
-                        }
-                    } else {
-                        this.setAttribute("course", compass.toString());
-                        this.setAttribute("course_deviation", "0");
+            const compass = Number(this.getAttribute('rotation'));
+            let displayCourseDeviation = false;
+            let displayVerticalDeviation = false;
+            if (this.navigationMode == Jet_NDCompass_Navigation.ILS || this.navigationMode === Jet_NDCompass_Navigation.VOR) {
+                let beacon;
+                if (this.navigationMode == Jet_NDCompass_Navigation.ILS) {
+                    beacon = this.gps.radioNav.getBestILSBeacon();
+                } else if (this.navigationMode === Jet_NDCompass_Navigation.VOR) {
+                    beacon = this.gps.radioNav.getBestVORBeacon();
+                }
+                if (beacon.id > 0) {
+                    displayCourseDeviation = true;
+                    let deviation = (SimVar.GetSimVarValue("NAV CDI:" + beacon.id, "number") / 127);
+                    const backCourse = SimVar.GetSimVarValue("AUTOPILOT BACKCOURSE HOLD", "bool");
+                    if (backCourse) {
+                        deviation = -deviation;
```

Comment:
I don't really understand this part.

Reply:
Is Asobo code from the changes they made in SU3... I don't know what the deviation does so I didn't alter their code.